### PR TITLE
Update install notes for Mizuki

### PR DESCRIPTION
**Pull request by danielballan**

### CODE REVIEW

File: source/fresh-installation.rst [hunk 1]
```diff
@@ -247,52 +285,52 @@ Add the following to the user's ``~/.bashrc`` file.
...
+should already be set at the system level but in practice they are often not.
 
 Conda has already been installed on all NSLS-II workstations (ws) and servers
 (srv) in a shared location. The last line adds conda to the user's PATH so that
 it overrides any system-installed Python, IPython, etc.
 
+Convenience Script ``bsui``
++++++++++++++++++++++++++++
+
+The script ``bsui``
```
Comment (stuartcampbell) [inline source/fresh-installation.rst:300]: Do we want to add a link to a copy of this ?

File: source/fresh-installation.rst [hunk 2]
```diff
@@ -22,9 +22,9 @@ Create Conda Environments
 
 A puppet class should install conda into ``/opt/conda`` on machines designated
 ``*-srv*`` ("server") or ``*-ws*`` ("workstation") as soon as they are on the
```
Comment (ghost) [inline source/fresh-installation.rst]: Is it worth mentioning the few exceptions that conda lives on ca or gpu machines?

Comment (stuartcampbell) [reply source/fresh-installation.rst]: Added a sentence

**Comment by stuartcampbell** [comment]
Looks ok to me, I've created an issue #66 to remind us to stop using @danielballan branch when doctr has the fix.

**Pull request merged**